platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/expressions/Expression.java

```diff
@@ -3224,6 +3226,123 @@ public static Expression collator(Expression caseSensitive, Expression diacritic
     return new Expression("collator", new ExpressionMap(map));
   }
 
+  /**
+   * Returns formatted text containing annotations for use in mixed-format text-field entries.
+   * <p>
+   * To build the expression, use {@link #formatEntry(Expression, FormatOption...)}.
+   * <p>
+   * "format" expression can be used, for example, with the {@link PropertyFactory#textField(Expression)}
+   * and accepts unlimited numbers of formatted sections.
+   * <p>
+   * Each section consist of the input, the displayed text, and options, like font-scale and text-font.
+   * <p>
+   * Example usage:
+   * </p>
+   * <pre>
+   * {@code
+   * SymbolLayer symbolLayer = new SymbolLayer("layer-id", "source-id");
+   * symbolLayer.setProperties(
+   *   textField(
+   *     format(
+   *       formatEntry(
+   *         get("header_property"),
+   *         formatFontScale(2.0),
+   *         formatTextFont(new String[] {"DIN Offc Pro Regular", "Arial Unicode MS Regular"})
+   *       ),
+   *       formatEntry(concat(literal("\n"), get("description_property")), formatFontScale(1.5)),
```

Comment:
This code example looks like it abruptly ends after `formatFontScale(1.5)),`, is that intentional?

Reply:
Nice catch :+1: